arrMethodsLab.js

```diff
@@ -0,0 +1,409 @@
+// # Exercises
+//
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+console.log("---------- #1 ----------")
+let arr1 = [1, 2, 3, 4, 5]
+let plusTen = []
+
+arr1.forEach((currentElem) => {
+  plusTen.push(currentElem + 10)
+}
+)
+
+console.log(plusTen)
+
+// ```
+// Input: [1,2,3,4,5]
+// Output: [11,12,13,14,15]
+// ```
+console.log(" ")
+
+// 2. Given an array, return a new array where every non-string element is converted to an empty string.
+console.log("---------- #2 ----------")
+
+let arr2 = ["a", 123, "b", "c", {name: "cat"}]
+let filteredArr = arr2.filter((currentElem) => {
+  if (typeof currentElem === "string") {
+    return currentElem
+  } else {
+    return ""
+  }
+})
+
```

Comment:
This first part is all thats needed . If it was wrapped in a function all you would have to do is return filteredArr.